app/src/main/res/drawable/ic_mic_none_white_24dp.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The name of the file is misleading. It's has `white` in the name when the color is not white.